mkdocs.yml

```diff
@@ -0,0 +1,16 @@
+site_name: Siddhi-Store-Elasticsearch
+extra_css:
```

Comment:
Needs to be changed to support mkdocs theme update

Reply:
Added the fixes: a3ce67ba98728e883dcd53baf7f40b92ee61e788

Reply:
adec20ea2c6499ccab87d812cb6a1dd0bc18c438